packages/firestore/src/platform/browser/webchannel_connection.ts

```diff
@@ -208,7 +209,9 @@ export class WebChannelConnection extends RestConnection {
     }
 
     if (this.useFetchStreams) {
-      request.useFetchStreams = true;
+      // When b/307942499 is fixed, revert to the following line:
+      // request.useFetchStreams = true;
```

Comment:
Remove commented code (line 213). This can be easily found in commit history (and/or add a comment about it in b/307942499)

Reply:
resolved